app/Mail/InviteUser.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace App\Mail;
+
+use App\Models\User;
+use App\Models\Invitation;
+use Illuminate\Bus\Queueable;
+use Illuminate\Mail\Mailable;
+use Illuminate\Queue\SerializesModels;
+use Illuminate\Contracts\Queue\ShouldQueue;
+
+class InviteUser extends Mailable
+{
+    use Queueable, SerializesModels;
+
+    /**
+     * Create a new message instance.
+     *
+     * @return void
+     */
+    public function __construct(User $user, $password)
+    {
+        $this->invitation = $user;
+        $this->password = $password;
+    }
+
+    /**
+     * Build the message.
+     *
+     * @return $this
+     */
+    public function build()
+    {
+        return $this->from('lucia@stats4sd.org')
```

Comment:
I feel automated invitation emails shouldn't come from an individual - it might be fine right now with at most a few users, but if this scales, I doubt you'd want dozens or hundreds of people all emailing you directly when they forget their passwords... 

I've setup a config('mail.admin_email') entry, which uses the ADMIN_EMAIL_ADDRESS env variable, or a default of support@stats4sd.org. If you want to keep the invites going from your personal email for now while you provide close support for the first few people using the platform, then leave this for now. But I suggest that at some point you swap this out for the config('mail.admin_email') so that we can both access any support requests that come from the platform.

Reply:
I think using the support@stats4sd.org is the best idea. I used mine, just to test the emails.